services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,21 +44,57 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
-  return ERR_CODE_SUCCESS;
+  if (event == NULL){
+    return ERR_CODE_INVALID_ARG;
+  }
+  if (thermalMgrQueueHandle == NULL) {
+    return ERR_CODE_INVALID_STATE;
+  }
+  if(xQueueSend(thermalMgrQueueHandle, event, portMAX_DELAY) == pdTRUE){
```

Comment:
this is too long to wait for queue to become lower- consider shortening it to a more reasonable value